src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -225,6 +227,10 @@ export default class OrgUnitsSelector extends React.Component {
         this.setState(state => ({
             roots: state.roots.map(r => (r.path === root.path ? mergeChildren(r, children) : r)),
         }));
+
+        if (this.props.onChildrenLoaded) {
+            this.props.onChildrenLoaded(children);
```

Comment:
The root org unit is not notified, right? I guess for a first approach (and for use case) this is not a problem.